api/envoy/extensions/transport_sockets/quic/v3/quic_transport.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package envoy.extensions.transport_sockets.quic.v3;
+
+import "envoy/extensions/transport_sockets/tls/v3/tls.proto";
+
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.transport_sockets.quic.v3";
+option java_outer_classname = "QuicTransportProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: QUIC_TRANSPORT]
```

Comment:
Why all caps?

Reply:
Doesn't need to be. Changed to lower case.